src/compiler/JSCodeGenVisitor.js

```diff
@@ -84,6 +117,24 @@ module.exports = class JSCodeGenVitor {
       this._out(`${cmd.variableName} = ${exp};`);
     } else if (cmd instanceof VariableBinding.End) {
       // nop
+    } else if (cmd instanceof FunctionBlock.Start) {
+      this._inFunctionBlock = true;
+      this._lastIndentLevel = this._indentLevel;
+      this.setIndent(0);
+
+      const exp = ExpressionFormatter.format(cmd.expression);
+      const functionName = `_template_${exp}`;
+      this._out(`template('${exp}', function ${functionName}() {`);
+      this.indent();
+
+      cmd.arguments.forEach((arg) => {
+        this._out(`const ${arg} = arguments[0]['${arg}'] || '';`);
```

Comment:
This satisfiers the requirement that missing options, when calling a template, should be treated as empty strings.